patterns/nagios

```diff
@@ -55,8 +55,13 @@ NAGIOS_EC_DISABLE_HOST_CHECK DISABLE_HOST_CHECK
 NAGIOS_EC_ENABLE_HOST_CHECK ENABLE_HOST_CHECK
 NAGIOS_EC_PROCESS_SERVICE_CHECK_RESULT PROCESS_SERVICE_CHECK_RESULT
 NAGIOS_EC_PROCESS_HOST_CHECK_RESULT PROCESS_HOST_CHECK_RESULT
-NAGIOS_EC_SCHEDULE_SERVICE_DOWNTIME SCHEDULE_SERVICE_DOWNTIME
+NAGIOS_EC_SCHEDULE_SVC_DOWNTIME SCHEDULE_SVC_DOWNTIME
```

Comment:
Following the existing naming the name is taken straight from the logs where this particular 'nagios_command' is defined as SVC instead of SERVICE in this instance.